bsonnumpy/bson/bson-config.h

```diff
@@ -142,14 +183,28 @@
 # undef BSON_HAVE_SYSCALL_TID
 #endif
 
-#define BSON_HAVE_RAND_R 0
+/*
+ * Define to 1 if _Decimal128 (BID format) is available on your platform.
+ */
+#define BSON_HAVE_DECIMAL128 0
+#if BSON_HAVE_DECIMAL128 != 1
+# undef BSON_HAVE_DECIMAL128
```

Comment:
We can remove BSON_HAVE_DECIMAL128. It's no longer present in libbson 1.17.0

Reply:
Done.